raku-advent-2020/local-drafts/p6steve-rudolph-on-raku.md

```diff
@@ -0,0 +1,183 @@
+# Rudolph on Raku
+
+## Finding a way home to the North pole with Physics::Navigation
+
+So, Rudolph has been worried about getting Santa and the other reindeer back home to the North Pole 
+after an exhausting flight to visit all the (well-behaved) Children on the Globe.
+
+He has heard a rumour that the North Pole keeps moving due to the precession of molten iron at 
+the Earth's core and that every year it creeps around a bit with relation to Santa's workshop,
+which lies at the True North Pole.
+
+Luckily he has been on a navigation skills course and has learned about how to specify a position on
+the globe using a combination of Latitude and Longitude. However, these seem all of a muddle as they 
+are alike and yet different. What Rudi needs is a way to structure his navigation to ensure that he 
+does not mix them up. Even better, he is good friends with Larry and knows that he can trust the 
+Raku type system to get him home.
```

Comment:
Add links to Raku, to type system, and most every Raku concept, if possible.

Reply:
I have done this where I can - since I am showing real code, there is quite a high density of Raku specific things, so I have been a bit selective to avoid death by explanation

Reply:
Well, a link to type system wouldn't be that kind of thing, and it would help newcomers to understand what Raku type system is about. Take into account that people arrive here from any kind of place, and they might not even know what's Raku. So, the more links, the merrier (christmas)